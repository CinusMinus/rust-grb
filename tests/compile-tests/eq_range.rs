use gurobi::*;
mod utils;

fn main() -> Result<()> {
  let _g = gag::Gag::stdout();
  create_model!(_g, m, x, y, z);
  c!(x + y + 2*z + 1 in 0..=1);
  c!(x + y + 2*z + 1 in ..=1);
  Ok(())
}
