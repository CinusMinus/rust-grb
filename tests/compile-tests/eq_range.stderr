error: Use '..' for range constraints
 --> $DIR/eq_range.rs:7:26
  |
7 |   c!(x + y + 2*z + 1 in 0..=1);
  |                          ^^^

error: Use '..' for range constraints
 --> $DIR/eq_range.rs:8:25
  |
8 |   c!(x + y + 2*z + 1 in ..=1);
  |                         ^^^
